packages/styles/scss/components/masthead/_masthead-l1.scss

```diff
@@ -0,0 +1,136 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '~carbon-components/scss/globals/scss/vars';
+@import '~carbon-components/scss/globals/scss/helper-mixins';
+@import '~carbon-components/scss/globals/scss/layout';
+@import '~carbon-components/scss/globals/scss/vendor/@carbon/elements/scss/import-once/import-once';
+@import '~carbon-components/scss/globals/scss/css--reset';
+
+/// @access private
+/// @group dotcom ui-shell Masthead L1
+
+$search-transition: cubic-bezier(0.2, 0, 0.38, 0.9);
+$search-transition-timing: 95ms;
+
+@mixin masthead-l1 {
+  .#{$prefix}--masthead__l1 {
+    display: flex;
+    width: 100vw;
+    height: 80px;
+    background-color: $ui-01;
+    transition-timing-function: $search-transition;
+  }
+
+  .#{$prefix}--masthead__l1-name {
+    display: flex;
+    height: 100%;
+    width: rem(336px);
+    background-color: $carbon--gray-90;
+    color: $text-04;
+    padding: $spacing-05;
+    align-items: flex-start;
+    flex-direction: column;
+
+    .#{$prefix}--masthead__l1-name-eyebrow {
+      a {
+        @include type-style('label-01');
+        color: $active-ui;
+        text-decoration: none;
+
+        &:hover {
+          text-decoration: underline;
+          transition: $search-transition-timing;
+        }
+      }
+
+      svg {
+        position: relative;
+        top: 4px;
+        left: -2px;
+        fill: $active-ui;
+      }
+
+      &:hover {
+        a {
+          color: $inverse-support-04;
+        }
+
+        svg {
+          fill: $inverse-support-04;
+        }
+      }
+    }
+
+    .#{$prefix}--masthead__l1-name-title {
+      position: relative;
+      padding: 0 $spacing-05;
+      margin-top: auto;
+      @include type-style('productive-heading-02');
```

Comment:
We just need to remember to update these when the 16px variants come through. For the time being, I put temporary component level variables in our global var file in hopes to ease this when the time comes.